src/Compilers/Core/Portable/Symbols/Attributes/CommonFieldWellKnownAttributeData.cs

```diff
@@ -119,5 +119,26 @@ public MarshalPseudoCustomAttributeData MarshallingInformation
             }
         }
         #endregion
+
+        #region NonNullTypesAttribute
+
+        private bool? _nonNullTypes;
+        public bool? NonNullTypes
```

Comment:
>public bool? NonNullTypes [](start = 8, length = 25)

Should these `NonNullTypes` properties be on the C# attribute data types only: `FieldWellKnownAttributeData`, etc.?